File: datasource/src/main/java/com/lrs/Bean/CovidBean.java
package com.lrs.Bean;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

//用来封装各省市疫情数据的javabean
@Data
@NoArgsConstructor
@AllArgsConstructor
public class CovidBean {
    private String provinceName;//省份
    private String provinceShortName;//省份短名
    private String cityName;//城市名称
    private Integer currentConfirmedCount;//当前确诊人数
    private Integer confirmedCount;//累积确诊人数
    private Integer suspectedCount;//疑似病例人数
    private Integer curedCount;//治愈人数
    private Integer deadCount;//死亡人数
    private Integer locationId;//位置ID
    private Integer pid;//位置ID
    private String statisticsData;//每一天的统计数据
    private String cities;//下属城市
    private String datetime;//时间

}

File: processs/src/main/scala/com/lrs/bean/CovidBean.scala
package com.lrs.bean

/**
 * @Author LRS
 * @Date 2022/7/12 18:09
 *       Desc
 */
case class CovidBean(
                      provinceName: String, //省份
                      provinceShortName: String, //省份短名
                      cityName: String, //城市名称
                      currentConfirmedCount: Integer, //当前确诊人数
                      confirmedCount: Integer, //累积确诊人数
                      suspectedCount: Integer, //疑似病例人数
                      curedCount: Integer, //治愈人数
                      deadCount: Integer, //死亡人数
                      locationId: Integer, //位置ID
                      pid: Integer,
                      statisticsData: String, //每一天的统计数据
                      cities: String, //下属城市
                      datetime: String //时间
                    )
